atlp-rwanda/Dev-Pulse-FN
# admin dashboard

#### What does this PR do?
Thie features in this PR are to be used by the admin to create authorized  emails
#### Description of Task to be completed?
This PR holds changes that will help the admin to create authorized emailsList and remove existing emails if needed

#### How should this be manually tested?
- clone the repo
- run yarn install
- checkout to `admin-dashboard`
- make sure your authorized email has the role of the manager
- Click on the admin button on the header the follow the onscreen instructions
- let me know for any feedback

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[]()
#### Screenshots (if appropriate)
##### landing on the admin dashboard
![image](https://user-images.githubusercontent.com/69194903/130244995-93c39d4e-6dd8-4fd4-b2ad-d7846eb2e0b6.png)
##### Clicking on Allowed emails
![image](https://user-images.githubusercontent.com/69194903/130245173-ed4c8632-2df1-4201-b623-73286f6ba488.png)
##### clicking on add emails or remove emails
![image](https://user-images.githubusercontent.com/69194903/130245338-0641dcf2-dffb-494c-8398-1202d8b5fc44.png)
##### adding emails
![image](https://user-images.githubusercontent.com/69194903/130245530-dd1e097b-0126-4f9c-8b46-e8e2dadbc942.png)
##### submitting the emals and are succussfully created
![image](https://user-images.githubusercontent.com/69194903/130245649-d6bbd37b-a3fc-483b-9815-def73367a3be.png)


#### Questions:
N/A
